testing/debug-accordion-ui.module.css

```diff
@@ -12,7 +12,7 @@
     flex-direction: row;
     align-items: center;
     gap: var(--wb-sizing-size_060);
-    background-color: var(--wb-semanticColor-core-foreground-inverse-default);
+    background-color: var(--wb-semanticColor-core-background-base-subtle);
```

Comment:
We can use a `background` token for the `background-color`, instead of a `foreground` token!